nni/retiarii/oneshot/pytorch/utils.py

```diff
@@ -180,3 +181,192 @@ def replace_input_choice(root_module, init_fn, modules=None):
         A list from layer choice keys (names) and replaced modules.
     """
     return _replace_module_with_type(root_module, init_fn, (InputChoice, nn.InputChoice), modules)
+
+def replace_module_with_type(root_module, init_fn, type_name, modules):
+
+    return _replace_module_with_type(root_module, init_fn, type_name, modules)
+
+class ParallelTrainValDataset(Dataset):
```

Comment:
Suggest deleting them.
If you are worried that you might need them in future. Keep them in a separate branch.